docs/src/content/docs/getting-started.mdx

```diff
@@ -115,7 +116,7 @@ If you are creating a theme inside an existing repository, you will have to set
 1. Run the playground to generate types for theme development and preview any changes
 
 	```sh
-	pnpm dev
+	pnpm playground:dev
```

Comment:
`playground:dev` is a custom script, because this is a manual setup we should assume that it doesn't exist and use `astro dev`

Reply:
Ah of course, what about suggesting `pnpm --filter playground dev`? When I was trying to figure out what to put here, I was trying to identify whether playground would always exist or not. If so, that command should work.

Reply:
Yes, that works too. Although it is not guaranteed for the name of the playground to be `playground`, but I am ok with using it. Another option could be to mention something like "navigate to the playground and then run dev".

Reply:
If folks start opening issues about it, we can change the language. For now I think this works well.